app/spike/internal/cmd/operator/restore.go

```diff
@@ -0,0 +1,80 @@
+//    \\ SPIKE: Secure your secrets with SPIFFE.
+//  \\\\\ Copyright 2024-present SPIKE contributors.
+// \\\\\\\ SPDX-License-Identifier: Apache-2.0
+
+package operator
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	spike "github.com/spiffe/spike-sdk-go/api"
+	"golang.org/x/term"
+
+	"github.com/spiffe/spike/app/spike/internal/trust"
+	"github.com/spiffe/spike/internal/auth"
+	"github.com/spiffe/spike/internal/log"
+)
+
+func newOperatorRestoreCommand(
```

Comment:
This and its "restore" counterpart require some cleanup.

I'll do that immediately after this PR.